docs/styles/styles.scss

```diff
@@ -2,4 +2,10 @@
 @use 'tools' as *;
 @use 'generic';
 @use 'base';
-@use 'chunks';
\ No newline at end of file
+@use 'chunks/layout';
+@use 'chunks/demo-embed';
+@use 'chunks/contributor';
+@use 'chunks/sr-only';
+@use 'chunks/heading-wrapper';
+@use 'chunks/external-link';
+@use 'chunks/site-footer';
```

Comment:
Trashed our `chunks/index` file, since we no longer export and load _all_ chunks from `styles.scss`! I also prefer mystery index files where possible, since direct imports are clearer to browse and update loater